apps/finance/test/finance.js

```diff
@@ -359,10 +359,11 @@ contract('Finance App', accounts => {
         })
 
         context('many accounting period transitions', () => {
-            let maxTransitions = 0
+            // Arbitrary number of max transitions to simulate OOG behaviour with transitionsPeriod
+            const maxTransitions = 20
 
             beforeEach(async () => {
-                maxTransitions = await app.MAX_PERIOD_TRANSITIONS_PER_TX().then(bn => bn.toNumber())
+                await app.mock_setMaxPeriodTransitions(maxTransitions)
```

Comment:
Do we need this mock function? Can we just set a big enough time forward and let it actually run out of gas?

Reply:
Wanted to do that, but was afraid of the fragility it would introduce into the tests (we'd have to guess how many transitions would force a OOG, and if it ever changed in the future, our tests could break).